packages/EasyCodingStandard/packages/SniffRunner/tests/Application/FileProcessorSource/SomeFile.php.inc

```diff
@@ -1,5 +1,16 @@
 <?php declare(strict_types=1);
 
-$array = array(
-    'someValue'
-);
+namespace Foo;
+
+final class SomeFile
+{
+    /**
+     * @var \Foo\Baz
+     */
+    private $foo;
+
+    /**
+     * @var \Foo\Baz
+     */
+    private $bar;
+}
```

Comment:
Add test case instead of removing one. This might cause regression.

Name it `ChangesetTest` to make it clear.

Reply:
It tests same thing which was tested before, no possible regression. It just tests additional things now

Reply:
Since the code has changed, it tests different code.

Reply:
It doesn't matter. Code before tested if processor detected there is one checker enabled. It doesn't matter if checker was changed.

Reply:
PR should be independent old tests while adding new feature, so I don't have to clone it localy see it passes.

I've met many anoying bugs caused by changing  tests  like this "it doesnt matter" way. Please separate it.

Reply:
Well test itself passes without changes, so you can see change of fixture don't have an effect on it. Will revert it and create new one though so we can move on

Reply:
Thanks